models/cart-discount/src/cart-discount/cart-discount-draft/presets/sample-data-fashion/employee-sale.ts

```diff
@@ -1,13 +1,15 @@
-import { CartDiscountValueRelativeDraft } from '@commercetools-test-data/cart-discount-value-relative';
 import { LocalizedString } from '@commercetools-test-data/commons';
 import {
   CustomerGroupDraft,
   TCustomerGroupDraft,
 } from '@commercetools-test-data/customer-group';
 import * as CartDiscountDraft from '../..';
-import { CartDiscountLineItemsTargetDraft } from '../../../..';
+import {
+  CartDiscountLineItemsTargetDraft,
+  CartDiscountValueRelativeDraft,
+} from '../../../..';
```

Comment:
I see that this is the same as it was previously, but not specifying the folder name at the end of the import makes me a lil nervous